utils/list-issues

```diff
@@ -21,9 +23,13 @@ cat << SRC > $FILE
      "\nDescription: " (lookup :body i)
      "\nState: " (if (eq? (lookup :state i) :open) "open" "closed")
      "\nLabels: " (show (lookup :labels i))
-     "\nComments:\n\n\n" (lines (map pretty-comment (lookup :comments i)))
+     "\nComments:\n\n\n" (unlines (map pretty-comment (lookup :comments i)))
 )))
 (put-str! (pretty-issue selected))
 SRC
-radicle -- $FILE | less
 
+if [ -z "$NIX_PATH" ]; then
```

Comment:
This can not reliably determine if Nix is used. For example, I have `NIX_PATH` set because I have it installed, but I’m not actively using it. Isn’t there something like `USING_NIX`?

Reply:
True. It's also not clear that we should *always* be passing these extra flags - it's possible someone is using NixOS but not our particular setup.

Shall I just set a flag in `nix/shell.nix`, and then check for it here?

Reply:
> Shall I just set a flag in `nix/shell.nix`, and then check for it here?

Sounds like a good idea.

Reply:
Done.